ThreeBotPackages/threefold/communityapp/actors/community_manager.py

```diff
@@ -26,7 +26,7 @@ def info_get_current_user(self, schema_out=None, user_session=None):
         :return:
         """
         cl = j.clients.freeflowpages.get()
-        user_id = cl.users.get_by_username(self.current_user)["id"]
+        user_id = cl.users.get_by_email(self.current_user)["id"]
```

Comment:
this can raise IndexError on None object or raise exception

Reply:
Fixed